early-return/src/start-workflow.ts

```diff
@@ -0,0 +1,34 @@
+import { Connection, Client, WithStartWorkflowOperation } from '@temporalio/client';
+import { transactionWorkflow } from './workflows';
+import { getTransactionConfirmation } from './shared';
+
+async function runTransactionWorkflow(transactionID: string, client: Client) {
+  const workflowId = 'transaction-' + transactionID;
+
+  const startWorkflowOperation = new WithStartWorkflowOperation(transactionWorkflow, {
+    workflowId,
+    taskQueue: 'early-return',
+    workflowIdConflictPolicy: 'FAIL',
+  });
+
+  const earlyConfirmation = await client.workflow.executeUpdateWithStart(getTransactionConfirmation, {
+    startWorkflowOperation,
+  });
+  const wfHandle = await startWorkflowOperation.workflowHandle();
+  const finalReport = await wfHandle.result();
```

Comment:
You should print the early confirmation before awaiting for the workflow completion. Right now, both will display at the same time, which fails to demonstrate the notion of an "early return".

Reply:
Done